infrastructure/README.md

```diff
@@ -10,9 +10,8 @@ Use `make` to run tasks to `create`, `start-private-net`, `stop-private-net`.
 If you want to create a local network without using our scripts then you can follow this [tutorial](https://developer.algorand.org/tutorials/create-private-network/**.
 
 
-## Quick start
-
-**TL;DR**: Make sure you have Algorand Node (`algod` and `goal`) and `make` installed. Then `cd infrastructure; make setup-private-net`. To reset use `recreate-private-net`.
+## Quick start (local development network)
```

Comment:
how about adding few lines about what are the benefits of using local, or sandbox ?
say, if you don't want to install algod, goal, proceed with docker

Reply:
It's already mentioned in the appropriate section - [https://github.com/scale-it/algo-builder/pull/379/files#diff-d5b352058053e1306240bcd8ab265b3234d611212cd196b2001fbb9131048566R53](https://github.com/scale-it/algo-builder/pull/379/files#diff-d5b352058053e1306240bcd8ab265b3234d611212cd196b2001fbb9131048566R53).
I prefer to keep it simple.